### DISCUSSION TITLE
Tissue concentration

### DISCUSSION THREAD

**Discussion by Ricardo-DO** [Open-Systems-Pharmacology/Forum#781] [Questions & Problems] [answered]
Hello everyone

I have a question, I was doing a simulation of an IV administration.
First I adjusted the model to the observed concentrations in blood (red dots and line) and it was fairly acceptable.

Liver and spleen were removed at 6h, homogenised and the compound quantified.

What I noticed is that the simulated concentration in "Tissue (observer)", for both liver and spleen, is lower than blood (liver, blue dot and line; spleen, green dot and line). 
At least with the liver, which is a highly vascularized tissue, the concentrations should be higher, right?

So my question is, what is considering the "Tissue observer" to calculate concentration?
To get the concentration as in the tissue homogenate, do I have to create an observer to sum the concentrations of intracellular, interstitial, plasma, RBC...?

Thanks in advance

Ricardo

![sim2](https://user-images.githubusercontent.com/59565610/93317496-7d5d1a80-f805-11ea-97ea-249aa8bc845d.png)

**Comment by StephanSchaller** [comment, marked as answer]
Dear @Ricardo-DO , tissue to plasma concentration ratio is governed by the plasma/tissue partitioning properties of the drug (depending on mostly lipophilicity of the drug). Perfusion is only a means of how fast this ratio might be reached but does not define it.

The "Tissue" observer provides the corresponding concentration from a BLOODLESS tissue homogenate (no blood or plasma).
If you still have blood left in the sample, use the "Whole Organ" observer and adjust the "Residual Blood Fraction for Sampling" to account for lost blood (bleeding out) in the sampling procedure.

Hope this helps.
Stephan

**Comment by prvmalik** [comment]
Just a note: The steep decline in your plasma concentration profile is caused by rapid uptake and sequestration in cells. It will be clearer to see when viewing the profile on a linear scale. For small molecule drugs, rapid uptake and sequestration can be driven by high lipophilicity, active transporters or binding to cellular components. One or more of these processes are missing in the model. Elimination is much too fast in your model. When you have updated the model accordingly, the tissue concentrations in the liver and spleen will be much closer to your observed data.

**Comment by andanteyang** [comment]
Hi I just have a quick question maybe not related to the issue. How do you calculate your tissue concentrations in ng/mL? Because all the tissue concentrations I got are in the unit of ng/g. 
Thanks in advance!

**Comment by prvmalik** [comment]
@andanteyang  In general we assume that the density of tissues is 1.00 - 1.04 g/mL

**Comment by Ricardo-DO** [comment]
Dear All

Thank you for your kind and illustrative replies.

Stephan, sorry I haven't replied before, I work on my job's laptop and I need to tell IT to update my PKSim version, I am still working with the v8. Are these options available on v9, right?

Paul, thanks for the advice. Yes, now we are thinking that there might be some active transport and sequestration or tissue binding as the clogP is around 0.5 and the clogD 7.4 ~-1. 
A weird compound, the Caco experiments show no permeation and I was thinking that it might be the same for tissues and it was the blood inside of them what it was making the concentration look high.

andanteyang, adding to Paul's comment. Yes, we homogenise our tissue with water for quantification and then we assume the tissue density as 1 g/mL.

Thanks!
Ricardo
